src/main/kotlin/blackjack/model/CardDeck.kt

```diff
@@ -0,0 +1,20 @@
+package blackjack.model
+
+class CardDeck {
+    var cards: List<Card> = INIT_CARDS
+        private set
+
+    fun drawCard(): Card {
+        val pickedCard = cards.shuffled().toMutableList()[0];
+        cards = cards.filter { it != pickedCard }
```

Comment:
```suggestion
        cards = cards.drop(1)
```
역시나 함수형 표현을 사용하면 조금 더 간단하게 표현할 수 있습니다. 코틀린에서 제공하는 이런저런 함수들에 대해서 학습해보셔도 좋을 것 같아요.